concepts/index.md

```diff
@@ -3,63 +3,101 @@ layout: single
 title:  "Concepts"
 sidebar:
   nav: concepts
-redirect_from: 
+redirect_from:
   - /docs
   - /docs/
   - /docs/overview
 ---
 
 {% include toc %}
 
-Spinnaker is an open source, multi-cloud continuous delivery platform that helps you release software changes with high velocity and confidence.
+Spinnaker is an open-source, multi-cloud continuous delivery platform that helps
+you release software changes with high velocity and confidence.
 
-It provides two core sets of features: *cluster management* and *deployment management*. Here is an overview of these features:
+Spinnaker provides two core sets of features:
 
-## Cluster management
+* [application management](#application-management)
 
-You use Spinnaker's cluster management features to view and manage your resources in the cloud.
+* [application deployment](#application-deployment)
+
+This article is an overview of these features:
+
+## Application management
+
+You use Spinnaker's application management features to view and manage your
+resources in the cloud environment.
+
+[Applications](#application), [clusters](#cluster), and [server
+groups](#server-group) are the key concepts Spinnaker uses to describe your
+services. Load balancers and firewalls describe how your services are exposed to
+users.
 
 ![](clusters.png)
 
-### Server Group
+### Application
 
-The base resource, the *Server Group*, identifies the deployable artifact (VM image, Docker image, source location) and basic configuration settings such as number of instances, autoscaling policies, metadata, etc. This resource is associated with a Load Balancer and a Firewall. When deployed, a Server Group is a collection of instances of the running software (VM instances, Kubernetes pods).
+An application in Spinnaker is a collection of clusters, which in turn are
+collections of server groups. The application also includes firewalls and load
+balancers.
+
+An application *represents* the service which you are going to deploy using
+Spinnaker, all configuration for that service, and all the infrastructure on
+which it will run.
+
+You will typically create a different application for each service, though
+Spinnaker does not enforce that.
 
-Server groups follow the **application-stack-detail-version** naming convention.
+Read more about applications, and how to create and configure them,
+[here](/guides/user/applications/).
 
 ### Cluster
 
-You can define *Clusters*, which are logical groupings of Server Groups in Spinnaker. These 
-groups are based on the **application-stack-detail** naming convention.
+You can define *Clusters*, which are logical groupings of Server Groups in Spinnaker. These
+groups are based on the [**application-stack
+-detail**](#the-application-stack-detail-naming-convention) naming convention.
 
-### Applications
+> **Note:** cluster, here, is not the same thing as a Kubernetes cluster.
```

Comment:
Addressed, sort of. Rather than explaining how they're different, I just tried to make it clear that one does not map to the other. The technical differences are somewhat beside the point.